### PR TITLE
Add refresh to front page

### PR DESCRIPTION
**Pull request by Vohveli**
Closes https://github.com/City-of-Helsinki/harrastuspassi-ui-android/issues/70

### CODE REVIEW

**Comment by Vohveli** [comment]
Ref: https://github.com/City-of-Helsinki/harrastuspassi-ui-android/issues/70

File: app/src/main/java/fi/haltu/harrastuspassi/fragments/HobbyEventListFragment.kt [hunk 1]
```diff
@@ -3,7 +3,6 @@ package fi.haltu.harrastuspassi.fragments
 import android.annotation.SuppressLint
 import android.app.ActivityOptions
 import android.content.Intent
-import android.content.res.ColorStateList
```
Comment (Doatti) [inline app/src/main/java/fi/haltu/harrastuspassi/fragments/HobbyEventListFragment.kt:6]: No code cleaning/refactoring in pull requests containing a new feature. Please make a separate pull request.

File: app/src/main/res/layout/fragment_home.xml [hunk 2]
```diff
@@ -1,24 +1,30 @@
...
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            android:theme="@style/ToolbarTitleStyles"
-            app:titleTextAppearance="@style/Toolbar.TitleText"/>
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent">
 
-    <ScrollView
+        <ScrollView
```
Comment (Doatti) [inline app/src/main/res/layout/fragment_home.xml:27]: This and the other white-space changes look a bit weird. Why is indentation added only to the first line of the ScrollView tag? Please check your Android Studio indentation settings.

**Review by Doatti — CHANGES_REQUESTED** [review]

**Review by Doatti — APPROVED** [review]

**Comment by Vohveli** [comment]
We've found a bug and we're going to fix it so don't accept pull request.

**Comment by Vohveli** [comment]
> We've found a bug and we're going to fix it so don't accept pull request.

Bug was that the app bar was missing when you scrolled down and now it's been fixed.
